<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>832</width>
    <height>519</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <property name="margin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="tabPosition">
       <enum>QTabWidget::West</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="T1">
       <attribute name="title">
        <string>ADC, DAC and INL</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QFrame" name="widgetFrameOuter">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <property name="class" stdset="0">
           <string>PeripheralCollection</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <property name="margin">
            <number>0</number>
           </property>
           <item>
            <widget class="QFrame" name="frame">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <layout class="QHBoxLayout" name="WidgetLayout"/>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QFrame" name="frame_2">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>50</height>
                </size>
               </property>
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <property name="class" stdset="0">
                <string>PeripheralCollectionInner</string>
               </property>
               <layout class="QGridLayout" name="gridLayout_2">
                <property name="horizontalSpacing">
                 <number>2</number>
                </property>
                <property name="margin">
                 <number>2</number>
                </property>
                <item row="1" column="0">
                 <widget class="QScrollArea" name="scrollArea">
                  <property name="widgetResizable">
                   <bool>true</bool>
                  </property>
                  <widget class="QWidget" name="scrollAreaWidgetContents">
                   <property name="geometry">
                    <rect>
                     <x>0</x>
                     <y>0</y>
                     <width>655</width>
                     <height>450</height>
                    </rect>
                   </property>
                   <layout class="QGridLayout" name="gridLayout_4">
                    <property name="margin">
                     <number>2</number>
                    </property>
                    <property name="spacing">
                     <number>2</number>
                    </property>
                    <item row="0" column="0">
                     <layout class="QGridLayout" name="plot_area"/>
                    </item>
                   </layout>
                  </widget>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QFrame" name="LegendFrame">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>100</width>
                 <height>0</height>
                </size>
               </property>
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <property name="class" stdset="0">
                <string>PeripheralCollectionInner</string>
               </property>
               <layout class="QVBoxLayout" name="LegendLayout">
                <property name="spacing">
                 <number>2</number>
                </property>
                <property name="margin">
                 <number>2</number>
                </property>
                <item>
                 <widget class="QListWidget" name="listWidget">
                  <property name="maximumSize">
                   <size>
                    <width>120</width>
                    <height>16777215</height>
                   </size>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="pushButton">
                  <property name="text">
                   <string>Crop</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="T2">
       <attribute name="title">
        <string>CAP, PCS, SEN</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QFrame" name="frame_7">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>650</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <property name="class" stdset="0">
           <string>PeripheralCollection</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>5</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>2</number>
           </property>
           <item row="0" column="1">
            <widget class="QFrame" name="Frame_4">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="PeripheralCollectionInner" stdset="0">
              <string/>
             </property>
             <property name="class" stdset="0">
              <string>PeripheralCollectionInner</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_5">
              <property name="sizeConstraint">
               <enum>QLayout::SetNoConstraint</enum>
              </property>
              <property name="margin">
               <number>0</number>
              </property>
              <property name="spacing">
               <number>0</number>
              </property>
              <item row="0" column="0" colspan="2">
               <widget class="Line" name="line">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
              <item row="1" column="0" colspan="2">
               <layout class="QVBoxLayout" name="tableLayout"/>
              </item>
             </layout>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QFrame" name="frame_4">
             <property name="maximumSize">
              <size>
               <width>120</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <property name="spacing">
               <number>3</number>
              </property>
              <property name="margin">
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>Click to Calibrate</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="ButtonSock">
                <property name="text">
                 <string>Socket Cap</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="Button330">
                <property name="text">
                 <string>330pF</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="Button680">
                <property name="text">
                 <string>680pF</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="Button2220">
                <property name="text">
                 <string>2220pF</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="ButtonPCS">
                <property name="text">
                 <string>PCS(0.5,1,1.5mA)</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="resistance">
                <property name="font">
                 <font>
                  <pointsize>8</pointsize>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="suffix">
                 <string> Ohms</string>
                </property>
                <property name="minimum">
                 <double>50.000000000000000</double>
                </property>
                <property name="maximum">
                 <double>10000.000000000000000</double>
                </property>
                <property name="value">
                 <double>1000.000000000000000</double>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="ButtonPCS_2">
                <property name="text">
                 <string>SEN</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="resistance_SEN">
                <property name="font">
                 <font>
                  <pointsize>8</pointsize>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="suffix">
                 <string> Ohms</string>
                </property>
                <property name="minimum">
                 <double>50.000000000000000</double>
                </property>
                <property name="maximum">
                 <double>10000.000000000000000</double>
                </property>
                <property name="value">
                 <double>1000.000000000000000</double>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="verticalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="ButtonReset">
                <property name="text">
                 <string>Reset Calibration</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="1" column="0" colspan="2">
            <layout class="QHBoxLayout" name="widgetLayout">
             <item>
              <widget class="QPushButton" name="pushButton_9">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximumSize">
                <size>
                 <width>70</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>RECOVER</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_8">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximumSize">
                <size>
                 <width>70</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>UPLOAD</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>832</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>listWidget</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>selected(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>794</x>
     <y>136</y>
    </hint>
    <hint type="destinationlabel">
     <x>875</x>
     <y>171</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>crop()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>824</x>
     <y>511</y>
    </hint>
    <hint type="destinationlabel">
     <x>842</x>
     <y>510</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_9</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>recover()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>301</x>
     <y>495</y>
    </hint>
    <hint type="destinationlabel">
     <x>358</x>
     <y>537</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_8</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>upload()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>552</x>
     <y>504</y>
    </hint>
    <hint type="destinationlabel">
     <x>554</x>
     <y>527</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ButtonReset</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>reset()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>131</x>
     <y>462</y>
    </hint>
    <hint type="destinationlabel">
     <x>151</x>
     <y>537</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ButtonSock</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>calSock()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>107</x>
     <y>100</y>
    </hint>
    <hint type="destinationlabel">
     <x>-16</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Button330</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>cal330()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>153</x>
     <y>134</y>
    </hint>
    <hint type="destinationlabel">
     <x>-18</x>
     <y>171</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Button680</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>cal680()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>165</x>
     <y>156</y>
    </hint>
    <hint type="destinationlabel">
     <x>-14</x>
     <y>203</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Button2220</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>cal2220()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>186</x>
     <y>194</y>
    </hint>
    <hint type="destinationlabel">
     <x>-15</x>
     <y>229</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ButtonPCS</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>calPCS()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>196</x>
     <y>237</y>
    </hint>
    <hint type="destinationlabel">
     <x>-33</x>
     <y>267</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ButtonPCS_2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>calSEN()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>156</x>
     <y>318</y>
    </hint>
    <hint type="destinationlabel">
     <x>-27</x>
     <y>318</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>run()</slot>
  <slot>delete_curve()</slot>
  <slot>selected(QString)</slot>
  <slot>crop()</slot>
  <slot>modify()</slot>
  <slot>recover()</slot>
  <slot>upload()</slot>
  <slot>reset()</slot>
  <slot>calPCS()</slot>
  <slot>cal330()</slot>
  <slot>cal680()</slot>
  <slot>cal2220()</slot>
  <slot>calSock()</slot>
  <slot>calSEN()</slot>
 </slots>
</ui>
